util/build.json

```
@@ -18,7 +19,8 @@
             "getopt.h",
             "pty.h",
             "execinfo.h",
-            ["file.h", "sys/file.h"]
+            ["file.h", "sys/file.h"],
+            "asprint.h"
```

```suggestion
            ["stdio.h", "stdio.h"]
```

asprintf and vasprintf are expected to be in [stdio.h](https://man7.org/linux/man-pages/man3/asprintf.3.html), so this header should be named the same.

Please copy what was done for flock in https://github.com/IBM/portlibfori/commit/a0a6b52e637ade3da99247356dd63f7bfffeade3#diff-630f36c8ca89eabc3e205055d1cfc267. You'll need to add the `#include_next` and we should add the `__asm__` bit too, to rename the symbols with the libutil_ prefix.